backend/app/services/translations/translate_project.rb

```diff
@@ -0,0 +1,50 @@
+module Translations
+  class TranslateProject
+    def initialize(project)
+      @project = project
+    end
+
+    def call
+      translate_content = TranslateContent.new(translatable_content, source_language_code)
+      (Language::TYPES - [source_language_code]).each do |target_language_code|
+        column_translations = translate_content.call(target_language_code)
+        present_translatable_columns.each.with_index do |col, idx|
+          I18n.with_locale(target_language_code) { @project.send(:"#{col}=", column_translations[idx]) }
+        end
+      end
+      @project.save!
+    end
+
+    def source_language_code
+      @project.language
+    end
+
+    def self.translatable_columns
```

Comment:
Hmm, I am not really convinced that we need special class for Project translation (TranslateProject). It would be great to have one class which translates every resource `Translations::Translate.new(resource).call`. Information about `translatable_columns` can be obtained from via Traco methods (I think method is called `translatable_attributes`). Rest of the code looks like it can be made more abstract